src/theme/components/newsletterSignup.ts

```diff
@@ -0,0 +1,77 @@
+import { getSectionColors } from "../../helpers/getSectionColors";
+import { SectionTypes } from "../../helpers/types";
+interface NewsLetterStyleProps {
+  newsletterSignupType: SectionTypes;
+}
+
+const NewsLetterSignup = {
+  baseStyle: ({ newsletterSignupType }: NewsLetterStyleProps) => ({
+    alignItems: "center",
+    borderWidth: { base: "0px 1px 1px 1px", md: "1px 1px 1px 0px" },
+    maxWidth: "1280px",
+    width: "100%",
+    "div#pitch": {
```

Comment:
Try using fewer ids and this can be `pitch: { ....`. Then in the component file, you can use `styles.pitch` explicitly instead of relying on the id css rule.

Reply:
Changed.